java/src/main/java/vw/jni/NativeUtils.java

```diff
@@ -88,7 +88,7 @@ else if (osName.toLowerCase().contains("linux")) {
             if (version == null) {
                 throw new UnsupportedOperationException("Cannot determine linux version");
             }
-            return distro.trim().replaceAll(" ", "_") + "." + version;
+            return (distro.trim().replaceAll(" ", "_") + "_" + version).toLowerCase();
```

Comment:
If you change the pattern here you have to change `build.sh` as well.  If you don't update this file at all then this change can be merged in without affecting `build.sh`.  I had intended the `.` to separate out different elements of the build environment in the name and `_` to denote word separation.  For example, in my opinion `vw_jni.Red_Hat.6.amd64.lib` is more legible than `vw_jni.Red_Hat_6_amd64.lib`.  As far as casing, this is designed to match the system calls to uname.  If you look at line 171 of `build.sh` you can see I specifically don't input `Darwin` but instead get the property directly.  I think this is more robust as I'm just reusing the system properties.